pkg/slsa/sign_test.go

```diff
@@ -145,7 +145,7 @@ P4amRng1j+1PnrdDixxQJtmAZT1lJZdXvQ==
 			bytes, err := attester.Sign(ctx, statement, signer)
 			require.NoError(t, err)
 
-			expected := formatPayload(`{"keyid":"some-ecdsa-key","sig":"MEUCIQClEWFrDoq/PelVgvqm2Tp5FEg62fYmi1bIYkTmctOQaAIgfXNOZBQxd+hXGsgKQsP/UyFCXInenAgJUUWuHgHu2LE="}`)
+			expected := formatPayload(`{"keyid":"some-ecdsa-key","sig":"MEUCIQCBPU7x0+KOBggqbfGGLFjsRilVD94yzOMPnDuAWhAVfgIgPDA5A2NwqPTo2ABJ/KNahJy38QfljO9GtuafK0RI5F0="}`)
```

Comment:
Changing the signature as the payload changed... this is expected, no?

Reply:
@chenbh do you happen to know?

Reply:
I acted too hastily. For whatever reason this fails for me locally. Is it a Mac / Linux thing?

Reply:
i had issues on a another pr with this test failing locally